frontend/packages/console-shared/src/components/popover-body/popover-body.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { DataPoint } from '@console/internal/components/graphs';
+import { ExternalLink, Humanize, resourcePathFromModel } from '@console/internal/components/utils';
+import { Dropdown } from '@console/internal/components/utils/dropdown';
+import { PodModel, NodeModel, ProjectModel } from '@console/internal/models';
+import { K8sKind } from '@console/internal/module/k8s';
+import { connectToURLs, MonitoringRoutes } from '@console/internal/reducers/monitoring';
+import { getPrometheusExpressionBrowserURL } from '@console/internal/components/graphs/prometheus-graph';
+import './popoverBody.scss';
+
+const dropdownKeys = ['By Pods', 'By Node', 'By Projects'];
+const models = [PodModel, NodeModel, ProjectModel];
+const dropdownItems = {
+  'By Pods': 'By Pods',
+  'By Node': 'By Node',
+  'By Projects': 'By Projects',
+};
+
+export const PopoverBody = connectToURLs(MonitoringRoutes.Prometheus)(
```

Comment:
this is specific component for dashboards/utilization, so it should go under `src/components/dashboard/utilization-card`